comp/remote-config/rcclient/rcclientimpl/rcclient.go

```diff
@@ -294,6 +294,14 @@ func (rc rcClient) agentConfigUpdateCallback(updates map[string]state.RawConfig,
 		err = rc.settingsComponent.SetRuntimeSetting("log_level", mergedConfig.LogLevel, model.SourceRC)
 	}
 
+	streamLogsSource := config.Datadog().GetSource("enable_stream_logs")
+
+	if mergedConfig.EnableStreamLogs != nil {
+		err = rc.settingsComponent.SetRuntimeSetting("enable_stream_logs", *mergedConfig.EnableStreamLogs, model.SourceRC)
```

Comment:
Assigning to `err` here discards any previous error we may have received on line 293. Is this intended?